src/hotspot/share/runtime/os.cpp

```diff
@@ -1180,7 +1180,8 @@ bool is_pointer_bad(intptr_t* ptr) {
 
 // Looks like all platforms can use the same function to check if C
 // stack is walkable beyond current frame.
-// Returns false if this is the cas
+// Returns true if this is not the case, i.e. the frame is possibly
```

Comment:
This comment sounds wrong. Surely we return true if it is the case that the given frame is the first C frame on the stack?

Reply:
Never mind I see the full context now. Would have been better to not start this comment on a new line in the original code.